recipes/gw/build.sh

```diff
@@ -5,9 +5,10 @@ if [[ "$OSTYPE" != "darwin"* ]]; then
   sed -i 's/-lEGL -lGLESv2/-lGL/' Makefile
 fi
 make prep
-CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" make
+CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" LDFLAGS="${LDFLAGS} -L${PREFIX}" prefix="${PREFIX}"  make -j ${CPU_COUNT}
```

Comment:
```suggestion
CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" LDFLAGS="${LDFLAGS} -L${PREFIX}/lib" prefix="${PREFIX}"  make -j ${CPU_COUNT}
```
But since the build already passes I think you don't need to add it explicitly at all.

Reply:
If we could get this merged as is, that would be great. then I can resolve some of these issues in an update build